smart-cards-tests/src/main/java/gov/va/api/health/smartcards/tests/SystemDefinitions.java

```diff
@@ -122,8 +122,16 @@ static SystemDefinition systemDefinition() {
 
     @NonNull String apiPath;
 
+    String formattedPort() {
+      if (port == 443 || port == 80) {
+        return "";
+      }
+      return String.format(":%s", port());
+    }
+
     String urlWithApiPath() {
       StringBuilder builder = new StringBuilder(url());
+      builder.append(formattedPort());
```

Comment:
There are other conditionals in this method, so I think we can inline this check too:
```
if (url().contains("localhost")) {
  builder.append(":" + port());
}
```

Reply:
While this would cover all current cases, I'd insist on the conditional being based on the port, rather than a 'localhost' URL:
```
if(port == 443 || port == 80) {
  builder.append(":" + port());
}
```
Technically speaking, this should also be checking for `http` vs `https`, but that's a bit of overkill.

Any preferences?

Reply:
I'd rather not hardcode 8096 here. I'd consider this one a magic number.

443 and 80 are the standard default ports for https and http. So a request to `https://api.va.gov/foo` is really being sent to `https://api.va.gov:443/foo` behind the scenes.

We could just remove the check- depending on how the request is being sent behind the scenes. I think our `doGet` method always includes the port, so it would be consistent.